presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -971,6 +977,28 @@ private String formatDomain(Domain domain)
         }
     }
 
+    private static String getFrameInfo(Window window)
+    {
+        if (!window.getFrame().isPresent()) {
+            return "";
+        }
+
+        WindowFrame frame = window.getFrame().get();
+        StringBuilder builder = new StringBuilder(frame.getType().toString());
+        if (frame.getStart().getValue().isPresent()) {
```

Comment:
Extract `frame.getStart()` into a variable. It will make the code easier to read, since it's used in 3 different places.